.github/workflows/build_docker_on_release.yml

```diff
@@ -59,3 +74,14 @@ jobs:
             org.opencontainers.image.created=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.source'] }}
             org.opencontainers.image.revision=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.revision'] }}
             org.opencontainers.image.version=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
+
+      # TODO: check if we can exclude this commit from the changelog
```

Comment:
I believe we can do this by making a specific commit subject and updating the `.autorc` in the workflows to omit the commit.

Reply:
Actually @surchs would you mind changing the subject of the commit to something we can then add to `.autorc` in this PR?
I've already opened an issue in workflows to address it: https://github.com/neurobagel/workflows/issues/137

Reply:
Good idea, will do!